scripts/rfsuite/tasks/sensors/elrs.lua

```diff
@@ -187,13 +187,26 @@ local function decAccel(data, pos)
     return nil, pos
 end
 
+local function convertToCoordinate(value)
+    -- Divide the input by 10^7 to get the correct coordinate
+    local coordinate = value / 10000000
+    
+    return string.format("%.4f", coordinate)
+end
+
 local function decLatLong(data, pos)
     local lat, lon
     lat, pos = decS32(data, pos)
     lon, pos = decS32(data, pos)
-    setTelemetryValue(0x1125, 0, 0, 0, UNIT_GPS, 0, "GPS")
-    setTelemetryValue(0x1125, 0, 0, lat, UNIT_GPS_LATITUDE)
-    setTelemetryValue(0x1125, 0, 0, lon, UNIT_GPS_LONGITUDE)
+ 
+    lat = convertToCoordinate(lat)
+    lon = convertToCoordinate(lon)
+    
+    lat = lat:gsub("%.", "")
+    lon = lon:gsub("%.", "")
```

Comment:
This looks especially dodgy.
Not sure what exactly it is trying to do, but converting a number to string with limited decimals, then removing decimal point, and converting back to a number sounds like a terrible hack

Reply:
Open to any suggestions really.

It was what I managed to come up with that at a minimum got to a summarised (short version) of the GPS coordinates.

Problem is this.

Frsky allows you to to to 4 decimal points (you have to send a whole number)

The issue is that the we are trying to get his GPS co-ordinate into a format that the custom sensor has no knowledge off.

Net result I hit a compromise where I limit the length of the long number so that the co-ordinate is effectively within the acceptable bounds and will still be prerty accurate on a map lookup.

Will see if I can find a cleaner way to mathematically do this.